packages/xrpl/src/utils/timeConversion.ts

```diff
@@ -45,7 +45,17 @@ function isoTimeToRippleTime(iso8601: string | Date): number {
   return unixTimeToRippleTime(isoDate.getTime())
 }
 
+/**
+ * Returns the current unix timestamp in seconds.
+ *
+ * @returns The current unix timestamp in seconds.
+ */
+function getUnixTimestamp(): number {
```

Comment:
Should we rename this to `getCurrentUnixTimestamp()` to avoid confusion? This might be mistaken into a conversion function

Reply:
Ah yes, we should.